iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_distributor.inl

```diff
@@ -87,10 +90,11 @@ ChunkDistributor<ChunkDistributorDataType>::removeQueue(cxx::not_null<ChunkQueue
 {
     typename MemberType_t::LockGuard_t lock(*getMembers());
 
-    auto iter = std::find(getMembers()->m_queues.begin(), getMembers()->m_queues.end(), queueToRemove);
+    const auto iter = std::find(getMembers()->m_queues.begin(), getMembers()->m_queues.end(), queueToRemove);
     if (iter != getMembers()->m_queues.end())
     {
-        getMembers()->m_queues.erase(iter);
+        /// @remark we don't use iter any longer so return value can be ignored
```

Comment:
Please use @ rationale as described in our CONTRIBUTING.md

Reply:
I tried @ rationale, My IDE did not recognize this as keyword and I also found no clear evidence that @ rationale is proper doxygen when searching the internet. So I came up with @ remark as a new proposal ;-)

Reply:
Then let's do it with the justification possibility the tool provides. Like this
`// PRQA S 3804 1 # we checked the capacity, so pushing will be fine`

Reply:
I'm unsure if this makes the code nicer, but it's a valid idea. `discardReturnValue` would be more explicit than the cast

Reply:
another possibility would be to hide the cast behind a macro, e.g. DISCARD_RESULT

Reply:
I don't remember the exact talk but remember "There's only one thing the C++ Community can agree on: Macros are evil" ;-)

Reply:
yes, macros are evil, but in cases like this they have their value and I'd rather take this macro than increasing the build time due to monomorphization of templates